chef_master/source/resource_zypper_package.rst

```diff
@@ -346,12 +347,9 @@ The following properties can be used to define a guard that is evaluated during
 
 Examples
 =====================================================
-.. tag resources_common_examples_intro
 
 The following examples demonstrate various approaches for using resources in recipes:
 
-.. end_tag
-
```

Comment:
I want to keep this dtag for the conversion to Hugo.

Reply:
I think that was the only one that was tagged. It was all over the other pages, but untagged there.

Reply:
It would be better to add it to the other pages, which I should have done when I first added that dtag. It might get yanked at a future date, but currently I add blocks of text to the resources pages in Hugo using dtags/shortcodes and that's one of them.

Reply:
I can get that added to the chef/chef rake task and then start rolling those out in a more consistent way int he future.